lib/grpnice.gi

```diff
@@ -301,13 +271,10 @@ GroupMethodByNiceMonomorphismCollColl( ClosureGroup,
 ##
 #M  ClosureGroup( <G>, <elm> )  . . . . . . . . closure of group with element
 ##
-# don't use `GroupMethodByNiceMonomorphismCollElm' to treat case of element
-# contained in.
-#GroupMethodByNiceMonomorphismCollElm( ClosureGroup,
-#    [ IsGroup, IsMultiplicativeElementWithInverse ] );
-InstallMethod(ClosureGroup,"by niceo",
-  IsCollsElms,[IsGroup and IsHandledByNiceMonomorphism,
-               IsMultiplicativeElementWithInverse],0,
+##  Don't use the generic code of `GroupMethodByNiceMonomorphismCollElm'
+##  to treat case of element membership.
+GroupMethodByNiceMonomorphismCollElm( ClosureGroup,
```

Comment:
The comment does not fit anymore?

Before `GroupMethodByNiceMonomorphismCollElm` indeed was not used here, now it is? Is this intentional, and why?